src/auto/css_section.rs

```diff
@@ -0,0 +1,58 @@
+// This file was generated by gir (e6cb5d0) from gir-files (11e0e6d)
+// DO NOT EDIT
+
+use CssSectionType;
+use ffi;
+use glib::translate::*;
+
+glib_wrapper! {
+    pub struct CssSection(Shared<ffi::GtkCssSection>);
+
+    match fn {
+        ref => |ptr| ffi::gtk_css_section_ref(ptr),
+        unref => |ptr| ffi::gtk_css_section_unref(ptr),
+    }
+}
+
+impl CssSection {
+
+    pub fn get_end_line(&self) -> u32 {
+        unsafe {
+            ffi::gtk_css_section_get_end_line(self.to_glib_none().0)
+        }
+    }
+
+    pub fn get_end_position(&self) -> u32 {
+        unsafe {
+            ffi::gtk_css_section_get_end_position(self.to_glib_none().0)
+        }
+    }
+
+    //pub fn get_file(&self) -> /*Ignored*/Option<gio::File> {
```

Comment:
I guess we'll have to generate `Gio.File` soon because there're no alternatives sometimes.